Source/Core/VideoCommon/Present.cpp

```diff
@@ -23,13 +23,73 @@
 
 std::unique_ptr<VideoCommon::Presenter> g_presenter;
 
+static constexpr int VIDEO_ENCODER_LMC = 4;
```

Comment:
Please add a general comment on what this is, I'm not sure what 'lmc' stands for for instance.